tensorflow/python/data/ops/iterator_ops.py

```diff
@@ -419,13 +420,14 @@ def get_next(self, name=None):
     if self._get_next_call_count > GET_NEXT_CALL_WARNING_THRESHOLD:
       warnings.warn(GET_NEXT_CALL_WARNING_MESSAGE)
 
-    # pylint: disable=protected-access
-    flat_ret = gen_dataset_ops.iterator_get_next(
+    with ops.device(self._iterator_resource.device):
+      # pylint: disable=protected-access
+      flat_ret = gen_dataset_ops.iterator_get_next(
         self._iterator_resource,
```

Comment:
you will need to fix the indent here

Reply:
these used to be indented by 4 spaces from the previous line before your change and now seem to be indented by 2 spaces

Reply:
You can't find all the lint warnings in https://source.cloud.google.com/results/invocations/eae2a68e-6a74-41aa-8e87-dfd6855e6249/log